observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,53 @@
+Configure these PVs to be used by Prometheus to store metrics data and by Alertmanager to store alert data. Specify the metrics data retention parameters for Prometheus and Thanos Ruler.
++
+[NOTE]
+====
+By default, a newly installed {product-title} system is configured to fire an alert named `PodStartupStorageOperationsFailing` to remind you that storage is not configured.
+You can resolve this alert on the *Alerts* tab in the *Observe* -> *Alerting* section of the {product-title} web console.
+====
++
+* If needed, for long-term data retention, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring_remote_write_storage_configuring-the-monitoring-stack[configure the remote write feature] to enable Prometheus to send ingested metrics to remote systems for storage.
+Be sure to xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#adding-cluster-id-labels-to-metrics_configuring-the-monitoring-stack[add cluster ID labels to metrics] for use with your remote write storage configuration.
+* If needed, xref:../../observability/monitoring/managing-alerts.adoc#managing-core-platform-alerting-rules_managing-alerts[modify or create alerting rules] for your cluster by defining alerting rules for different conditions based on metrics and labels.
+These rules specify the conditions that trigger alerts, such as high CPU or memory usage, network latency, and so forth.
+* xref:../../observability/monitoring/managing-alerts.adoc#sending-notifications-to-external-systems_managing-alerts[Configure alert receivers] so that Alertmanager can send alerts to an external notification system such as email, Slack, or PagerDuty.
+* If needed, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#managing-cpu-and-memory-resources-for-monitoring-components[specify resource limits and requests for monitoring components] to ensure that the containers that run monitoring components have enough CPU and memory resources.
+* xref:../../observability/monitoring/managing-metrics.adoc#managing-metrics[Configure service monitors], which define the set of services and endpoints that Prometheus scrapes for metrics.
+Create these `ServiceMonitor` resources for each service you want to monitor.
+These resources specify the labels that identify the target services and the scraping configuration, including the endpoint paths and HTTP ports to scrape.
+
+With the monitoring stack configured to suit your needs, Prometheus will start scraping metrics from the specified services and storing these metrics.
+You can now go to the *Observe* pages in the {product-title} web console to view and query collected metrics, manage alerts, identify performance bottlenecks, and scale resources as needed:
```

Comment:
I think a lot of this can be shortened because we add links to the sections that explain how to view and query metrics:
```suggestion
With the monitoring stack configured to suit your needs, Prometheus scrapes metrics from the specified services and stores these metrics. You can now view and query collected metrics, manage alerts, identify performance bottlenecks, and scale resources as needed:
```

Reply:
good one. changed